inst/pages/Introductory-workflow/Introductory.qmd

```diff
@@ -0,0 +1,293 @@
+
+## Assays
+
+An assay is a way of measuring the presence and abundance of different types of microbes in a sample. If you want to know how many bacteria of a certain type are in your gut, you can use an assay to measure this. In this example, we subset the assay using brackets to show only the first 5 rows and the first ten columns. This makes it easier to understand and read. We'll discuss more on subsetting later.
+
+```{r}
+assay(tse)[1:5,1:10] 
+```
+
+## colData
+
+Another important aspect of microbiome analysis is sample data.
+
+```{r}
+tse_df_colData <- as.data.frame(colData(tse))
+datatable(tse_df_colData,options = list(pageLength = 5),rownames = FALSE)
+```
+
+In this example, `colData(tse)` returns a DataFrame with 26 rows and 7 columns. Each row represents a sample, while each column represents a different aspect of the sample data.
+
+## rowData
+
+rowData contains data on sample characteristics, particularly taxonomic information.
+
+```{r}
+#| echo: false 
+#| output: false
+options(warn = -1)
+```
+
+```{r}
+tse_df_rowData <- as.data.frame(rowData(tse))
+datatable(tse_df_rowData, options = list(pageLength = 5))
+```
+
+```{r}
+#| echo: false 
+#| output: false
+options(warn = 0)
+```
+
+Here `rowData(tse)` returns a DataFrame with 151 rows and 7 columns. Each row represents an organism and each column a taxonomic level.
+
+In the example, ***Kingdom*** indicates whether the feature belongs to the kingdom of bacteria or archaea, ***Phylum*** indicates the phylum to which the individual belongs and so on down to the specific species.
+
+To illustrate the structure of a *TreeSummarizedExperiment*, here's [**an article**](https://f1000research.com/articles/9-1246) that uses this type of object. Also, please take a look at figure 1 below.
+
+![1. Storing microbial data : the structure of a TreeSummarizedExperiment](images/TSE-structure.svg){.lightbox .contentimg}
+
+<hr>
+
+# Wrangling the data
+
+In some cases, you may need to change your data in order to get your wanted results. In this section we'll go over how to agglomerate data, subset your data and more. A *TreeSummarizedExperiment* allows for some neat data manipulation using [**the dplyr package**](https://dplyr.tidyverse.org/).
+
+## Subsetting
+
+In some cases, you may only need to use part of your original *TreeSummarizedExperiment*.
+
+Using the Tengeler2020 dataset, we can focus on a certain cohort for example. Doing this is pretty straightforward:
+
+```{r}
+tse_subset_by_sample <- tse[ , tse$cohort =="Cohort_1"]
+```
+
+This will create a *TreeSummarizedExperiment* Object only containing samples of the first cohort. To visualize the resulting object as a dataframe we can use the `meltAssay` method and specify what columns will be put into the dataframe:
```

Comment:
melting example could better go to the chapter in OMA that mentions meltAssay

Reply:
Also meltAssay is now meltSE